<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Brennen Raimer</author>
 <class>PowerSchool_DataViz</class>
 <widget class="QMainWindow" name="PowerSchool_DataViz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PowerSchool DataViz</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PowerSchool_DataViz_Resources.qrc">
    <normaloff>:/icons/icons/Round_Landmark_Icon_School.svg</normaloff>:/icons/icons/Round_Landmark_Icon_School.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Graph_Tab">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="Graph_Grid">
          <item row="0" column="0">
           <widget class="QWebEngineView" name="Graph_View">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Data_Table_Tab">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="Data_Table_Grid">
          <item row="0" column="0">
           <widget class="QTableWidget" name="Data_Table">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>0</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="File_Menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Graph_As"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_PowerSchool_DataViz"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="File_Menu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="PowerSchool_DataViz_Resources.qrc">
     <normaloff>:/icons/icons/baseline-folder_open-24px.svg</normaloff>:/icons/icons/baseline-folder_open-24px.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="PowerSchool_DataViz_Resources.qrc">
     <normaloff>:/icons/icons/baseline-exit_to_app-24px.svg</normaloff>:/icons/icons/baseline-exit_to_app-24px.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_PowerSchool_DataViz">
   <property name="text">
    <string>About PowerSchool DataViz</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset resource="PowerSchool_DataViz_Resources.qrc">
     <normaloff>:/icons/icons/baseline-open_in_new-24px.svg</normaloff>:/icons/icons/baseline-open_in_new-24px.svg</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_Graph_As">
   <property name="icon">
    <iconset resource="PowerSchool_DataViz_Resources.qrc">
     <normaloff>:/icons/icons/baseline-save_alt-24px.svg</normaloff>:/icons/icons/baseline-save_alt-24px.svg</iconset>
   </property>
   <property name="text">
    <string>Save Graph As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="PowerSchool_DataViz_Resources.qrc">
     <normaloff>:/icons/icons/baseline-print-24px.svg</normaloff>:/icons/icons/baseline-print-24px.svg</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PowerSchool_DataViz_Resources.qrc"/>
 </resources>
 <connections/>
</ui>
